Assets/Scripts/MiniGames/Song/text_scale.cs

```diff
@@ -0,0 +1,76 @@
+using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+using TMPro;
+
+
+public class text_scale : MonoBehaviour
```

Comment:
can you put a comment at the top of this file explaining what it's for? it's not quite clear to me from reading it

Reply:
lol I tough that would be clear, but yeah I had to switch the normal text out for text mesh

Reply:
actually, from what i can tell the script isn't even used. the text just changes color as it comes in rather than scaling right? if it is used, it's still worth commenting even if i remember what it's for since someone else who wasn't in all the meetings might work on it next